eng/pipelines/templates/stages/archetype-java-release-batch.yml

```diff
@@ -271,9 +271,10 @@ stages:
                 ArtifactPath: $(Pipeline.Workspace)/packages-signed
             - template: /eng/common/pipelines/templates/steps/create-apireview.yml
               parameters:
+                PackageInfoFiles:
+                  - ${{ each artifact in parameters.Artifacts }}:
```

Comment:
You are already doing the mapping from artifact to package info right here. You can do that for everyone and not need to bifurcate all the scripts.